src/App.scss

```diff
@@ -2,3 +2,6 @@
   margin: 0 auto;
 }
 
+.table-element{
+  text-align: center;
+}
```

Comment:
Remember, components are independent building blocks!! If you were to use your component outside of this app, it would break because you don't have the style in the component.
Move the style to a newly-created table.scss file and import it in the `Table.jsx`
```suggestion
```

Reply:
Ok, deleted, it actually works without scss now

Reply:
done